test/scopeSync.test.ts

```diff
@@ -177,7 +177,7 @@ describe('ScopeSync', () => {
       expect(setExtrasScopeSpy).toHaveBeenCalledExactlyOnceWith({ key: 'value', second: 'bar' });
     });
 
-    /* eslint-disable-next-line jest/no-focused-tests */
+    // eslint-disable-next-line @sentry-internal/sdk/no-focused-tests
```

Comment:
Jest flag was no longer available and it was blocking build so I just disabled the internal Sentry flag that has the same effect for this line